### ISSUE TITLE
Produce a "How to restore Bisq from a backup" guide

### ISSUE THREAD

**Issue by cbeams** [bisq-network/bisq-docs#18]
Based on information in the forum post at [1], write a tutorial that guides users through the process of restoring Bisq from a backup.

The path for this document should be https://github.com/bisq-network/docs/blob/master/exchange/howto/restore-from-backup.adoc, and should follow the format of other existing "howto" tutorials in https://github.com/bisq-network/docs/blob/master/exchange/howto.

See also [2] and [3].

[1] https://forum.bisq.io/t/switching-to-a-new-data-directory/1697
[2] https://forum.bisq.io/t/changing-computers/3365
[3] https://twitter.com/kryptopraxis/status/932664881685377025

**Comment by btcpb** [comment]
Hi, I humbly ask for some feedback on a video i made on how to restore bisq from backup https://www.youtube.com/watch?v=TK9oTwDPv1s. Thanks

**Comment by cbeams** [comment]
Closing as out of date. See the current index of https://docs.bisq.network where there is a placeholder entry for this doc. Anyone who wishes to write it is welcome to come discuss in the #bisq-docs channel in Slack, to submit a pull request, etc.

@btcpb, I did watch your video, and while the instructions appear to be accurate enough, the style, music and overall presentation of the information isn't a good match for Bisq.
